pkg/kubelet/cm/helpers_linux.go

```diff
@@ -223,7 +224,7 @@ func getCgroupSubsystemsV1() (*CgroupSubsystems, error) {
 
 		for _, subsystem := range mount.Subsystems {
 			previous := mountPoints[subsystem]
-			if previous == "" || len(mount.Mountpoint) < len(previous) {
+			if previous == "" || !strings.HasPrefix(previous, "/sys/fs/cgroup") && strings.HasPrefix(mount.Mountpoint, "/sys/fs/cgroup") || (len(mount.Mountpoint) < len(previous) && strings.HasPrefix(mount.Mountpoint, "/sys/fs/cgroup")) {
```

Comment:
can you add a comment on how we're doing the priority ranking here? I also feel like something like a switch statement or a differently structured if statement may be a bit clearer, idk